File: GCC-easy-webdeveloper13/csharp/Question1/Question1.cs
using System.CodeDom.Compiler;
using System.Collections.Generic;
using System.Collections;
using System.ComponentModel;
using System.Diagnostics.CodeAnalysis;
using System.Globalization;
using System.IO;
using System.Linq;
using System.Reflection;
using System.Runtime.Serialization;
using System.Text.RegularExpressions;
using System.Text;
using System;

namespace Solution
{
    class Solution
    {

        // You may change this function parameters
        static int FindMaxProfit(int numOfPredictedDays, int[] predictedSharePrices)
        {
            return -1;
        }

        static void Main(string[] args)
        {
            String[] firstLine = Console.ReadLine().Split(" ");

            int[] firstLineIntArr = Array.ConvertAll(firstLine, sTemp => Convert.ToInt32(sTemp));

            int numOfPredictedDays = firstLineIntArr[0];
            int[] predictedSharePrices = firstLineIntArr.Skip(1).ToArray();


            int result = FindMaxProfit(numOfPredictedDays, predictedSharePrices);

            // Do not remove this line
            Console.WriteLine(result);
            // Do not print anything after this line
        }
    }
}

File: GCC-easy-webdeveloper13/javascript/Question1/Question1.js
'use strict';

const fs = require('fs');

process.stdin.resume();
process.stdin.setEncoding('utf-8');

let inputString = '';
let currentLine = 0;

process.stdin.on('data', inputStdin => {
    inputString += inputStdin;
});

process.stdin.on('end', function() {
    inputString = inputString.replace(/\s*$/, '')
        .split('\n')
        .map(str => str.replace(/\s*$/, ''));

    main();
});

function readLine() {
    return inputString[currentLine++];
}

function findMaxProfit(numOfPredictedDays, predictedSharePrices){
    return -1;
}

function main() {
    const line = readLine().split(' ');
    const numOfPredictedDay = line[0];
    const predictedSharePrices = line.slice(1)
    const predictedSharePricesInt = predictedSharePrices.map(num => parseInt(num, 10));

    const result = findMaxProfit(numOfPredictedDay, predictedSharePricesInt);

    // Please do not remove the below line.
    console.log(result)
    // Do not print anything after this line
}
